help/sources/tutorials/ui/create/analytics/rainfocus-experience.md

```diff
@@ -0,0 +1,202 @@
+
+3. Select the **+** icon, in the bottom right corner, to create a new profile.
+
+4. Choose the **Adobe Experience Platform CDP** and click **OK**  
+![Screenshot showing the RainFocus platform and the location of the AEP CDP Integtaion Profile](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-select.png)
+
+5. Copy the following credential, previously created in the Adobe Developer Portal Project
+
+   1. **Client ID**
+   2. **Client Secret**
+   3. **Technical Account ID**
+   4. **Organization ID**
+
+6. Click **Save.** You should now see the new Integration Profile listed.
+
+7. Select the new Integration Profile and open it again.
+
+8. You will now see a list of predefined **push types** already configured. These are the [Experience Events](https://experienceleague.adobe.com/docs/experience-platform/xdm/classes/experienceevent.html?lang=en) that will be sent to AEP when they occur.   
+![Screenshot showing the RainFocus platform and the Experience Events that are configured automatically](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-setup.png)
+
+9. Select the **Sample JSON Payload** tab
+
+10. Highlight and copy the Sample JSON Payload and **save it in a new file with a .json extension**. We will use this in AEP to configure our mappings. 
+![Screenshot showing the RainFocus platform and the JSON preview that is generated](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-json.png)
+
+> **Setup is not yet complete** - _we will need to return to complete the integrations profile, keep this tab open and proceed to the next step to install and configure the Source Connector in AEP and to obtain the_ **_Streaming Endpoint_** _and_ **_Dataflow ID._**
+
+## Install and configure the RainFocus Source Connector
+
+1. Login into **Adobe Experience Platform**
+ 
+2. Locate **Sources** in the main navigation
+![Screenshot showing AEP and the location of Sources](/help/sources/images/tutorials/create/rainfocus/rainfocus_aep-sources.png)
+
+3. Select to the **Catalog** tab and search for **RainFocus** then click **Setup
+![Screenshot showing how to search for the RainFocus Source Connector](/help/sources/images/tutorials/create/rainfocus/rainfocus_sources-rf.png)
+
+4. You will be prompted to provide a **Sample JSON Payload** to configure mappings, upload the Sample JSON Payload downloaded and saved in the previous step.
+![Screenshot showing the JSON upload step of the RainFocus Source Connector](/help/sources/images/tutorials/create/rainfocus/rainfocus_source-json-upload.png)
+
+5. Once uploaded, you should see a visual representation of the payload, as shown below:
+![Screenshot showing the JSON preview of the uploaded sample](/help/sources/images/tutorials/create/rainfocus/rainfocus_source-json-preview.png)
```

Comment:
Please see template for Select Data and Preview data step: https://experienceleague.adobe.com/docs/experience-platform/sources/sdk/streaming-sdk/streaming-template-ui.html?lang=en#select-data